.github/workflows/clp-core-build.yaml

```diff
@@ -0,0 +1,48 @@
+name: Test Building CLP Core on Ubuntu Focal, Bionic and Centos 7.4
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - 'components/core/**'
+      - '.github/workflows/clp-core-build.yaml'
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'components/core/**'
+      - '.github/workflows/clp-core-build.yaml'
+  workflow_dispatch:
+
+jobs:
+  build-focal:
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/y-scope/clp/clp-core-dependencies-x86-ubuntu-focal:main
```

Comment:
Can we use `${{github.repository}}` (makes it easier to fork)?

Reply:
Are we sure we want to use that? Each user will first have to run the dependency worfklow in their fork to generate the dependency images before they use the core-build workflow. Which is useful if your'e making changes to the dependency images, but that will be infrequent and most of the time will cause unecessary overhead

Reply:
Oh good point. Sorry, didn't think it through.